error.md

```diff
@@ -0,0 +1,62 @@
+# Errors faced during migration with nikita v1
+
+## Installation of semver necessary
+
+When running the `bootstrap` command, I got an error stating that `semver` module was not found.
+
```

Comment:
who generates this error, if nikita-arch then it was probably the case before and should be fixed in master, if nikita then it shall be fixed in nikita, either case, I am sceptical the error belongs to here.

Reply:
This was raised by `nikita`.

Reply:
The open an issue on Nikita and fix it there, no need for a comment here.